components/gfx/display_list/mod.rs

```diff
@@ -839,19 +799,14 @@ pub struct SolidColorDisplayItem {
     pub color: Color,
 }
 
-impl HeapSizeOf for SolidColorDisplayItem {
-    fn heap_size_of_children(&self) -> usize {
-        self.base.heap_size_of_children()
-    }
-}
-
 /// Paints text.
-#[derive(Clone)]
+#[derive(Clone, HeapSizeOf)]
 pub struct TextDisplayItem {
     /// Fields common to all display items.
     pub base: BaseDisplayItem,
 
     /// The text run.
+    #[ignore_heap_size = "We exclude this because it is non-owning"]
```

Comment:
"Because it is non-owning" is probably enough for the comment? The "We exclude this" part is redundant once you know that the string is an explanation for the ignore.